<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sala2</class>
 <widget class="QMainWindow" name="sala2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>sala2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbImg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/sala2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbDesc">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>540</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(173, 56, 35);
font: 16pt &quot;Informal Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Encontre o objeto escondido...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btDesistir">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Desistir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btObj">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
